checks/sast.go

```diff
@@ -25,7 +25,7 @@ import (
 // CheckSAST is the registered name for SAST.
 const CheckSAST = "SAST"
 
-var sastTools = map[string]bool{"github-code-scanning": true, "lgtm-com": true, "sonarcloud": true}
```

Comment:
this assumes that the tool is installed an a GH App. Is this how it works? I thought it was a CLI users need to run. if so, we may need to wait for https://github.com/ossf/scorecard/pull/1487 to land.

Wdut?

Reply:
It is a GH Action. So will it be an issue? I don't think so.

Reply:
sorry I missed something when reading the PR. You can remove `tfsec` from the list. This list is only for GH apps.
The code `tfsecCheckDefinitions()` takes care of declaration of GH actions.